doc/user_guide/dynamo0p3.rst

```diff
@@ -772,16 +772,16 @@ then the permitted access modes depend on the type of data it is and
 the function space it is on. Valid values are given in the table
 below.
 
```

Comment:
Thanks Iva. It's looking a lot better but I think it would actually be nice (sorry) to have two tables: one for the valid access modes for user-supplied kernels and one for built-ins. That way it will be clear that user-supplied kernels cannot modify scalar arguments in any way. Possibly you could have two subsections, one for built-ins and one for user kernels but that might be clunky - I leave it to you to decide.

Reply:
I decided to do this properly and introduced "Valid Access Modes" in "Built-ins" chapter. Now each table lists all allowed accesses for the types of data allowed in user-defined Kernels and Built-ins. That means I took the scalar accesses for Built-ins out of the Kernel table.

Reply:
Thanks, that's better. Just a little tweaking required now.